examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -352,6 +360,20 @@ def parse_args(input_args=None):
         "'TOK,TOK2,TOK3' etc.",
     )
 
+    parser.add_argument(
+        "--base_new_tokens_off_token_abstractions",
+        default=False,
+        action="store_true",
+        help="utilize <TOK_0><TOK_1>... per each token instead of the default <si><si+1>..."
+    )
+
+    parser.add_argument(
+        "--base_new_tokens_off_token_abstractions",
+        default=False,
+        action="store_true",
+        help="utilize <TOK_0><TOK_1>... per each token instead of the default <si><si+1>..."
+    )
+
```

Comment:
I think this is a duplicate :)

Reply:
Thanks for noticing, the duplicate has been fixed!